app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -356,12 +356,19 @@ class SampleView extends React.Component {
           <ViewHeader.Controls>
             <BasicPopup
               trigger={<ShareButton onClick={this.onShareClick} />}
-              content="A shareable URL will be copied to your clipboard!"
+              content="A shareable URL was copied to your clipboard!"
               on="click"
               hideOnScroll
             />{" "}
             {/* TODO: (gdingle): this is admin-only until we have a way of browsing visualizations */}
-            {this.props.admin && <SaveButton onClick={this.onSaveClick} />}{" "}
+            {this.props.admin && (
```

Comment:
why not put the popup inside `SaveButton` ?

Reply:
Because one of instance requires a custom CSS class on the inner `Button`. I could add a prop `buttonClassName` and pass it down. Is that what you would do?

Reply:
You can just pass `className` and should work since props are passed downstream.
(We are actually creating a button, despite the fact that tooltip encloses the button, so the generic name className should apply to button).